specification/cognitiveservices/AnomalyDetector/univariate/models.tsp

```diff
@@ -44,150 +44,149 @@ enum TimeGranularity {
+"customInterval":5}.
 """)
   customInterval?: int32;
 
   @doc("""
-Optional argument, periodic value of a time series. If the value is null or
-does not present, the API will determine the period automatically.
+Argument that indicates the periodic value of a time series. If the value is null or
+is not present, the API determines the period automatically.
 """)
   period?: int32;
 
-  @doc("Optional argument, advanced model parameter, max anomaly ratio in a time series.")
+  @doc("Argument that indicates an advanced model parameter. It's the maximum anomaly ratio in a time series.")
   maxAnomalyRatio?: float32;
 
   @doc("""
-Optional argument, advanced model parameter, between 0-99, the lower the value
-is, the larger the margin value will be which means less anomalies will be
+Argument that indicates an advanced model parameter between 0 and 99. The lower the value
+is, the larger the margin value is, which means fewer anomalies will be
 accepted.
 """)
   sensitivity?: int32;
 
   @doc("""
-Used to specify how to deal with missing values in the input series, it's used
+Specifies how to deal with missing values in the input series. It's used
 when granularity is not \"none\".
 """)
   imputeMode?: ImputeMode;
 
   @doc("""
-Used to specify the value to fill, it's used when granularity is not \"none\"
+Specifies the value to fill. It's used when granularity is not \"none\"
 and imputeMode is \"fixed\".
 """)
   imputeFixedValue?: float32;
 }
 
-@doc("The definition of input timeseries points.")
+@doc("Definition of input time series points.")
 model TimeSeriesPoint {
-  @doc("Optional argument, timestamp of a data point (ISO8601 format).")
+  @doc("Argument that indicates the time stamp of a data point (ISO8601 format).")
   timestamp?: zonedDateTime;
 
-  @doc("The measurement of that point, should be float.")
+  @doc("Measurement of that point.")
   value: float32;
 }
 
-@doc("The response of entire anomaly detection.")
+@doc("Response of the entire anomaly detection.")
 model UnivariateEntireDetectionResult {
   @doc("""
-Frequency extracted from the series, zero means no recurrent pattern has been
+Frequency extracted from the series. Zero means no recurrent pattern has been
 found.
 """)
   period: int32;
 
   @doc("""
-ExpectedValues contain expected value for each input point. The index of the
+ExpectedValues contains the expected value for each input point. The index of the
 array is consistent with the input series.
 """)
   expectedValues: float32[];
 
   @doc("""
-UpperMargins contain upper margin of each input point. UpperMargin is used to
-calculate upperBoundary, which equals to expectedValue + (100 -
-marginScale)*upperMargin. Anomalies in response can be filtered by
-upperBoundary and lowerBoundary. By adjusting marginScale value, less
-significant anomalies can be filtered in client side. The index of the array is
+UpperMargins contains the upper margin of each input point. UpperMargin is used to
```

Comment:
Nit: almost no other docs in this PR reference the property name in the docs themselves. For this and LowerMargins, couldn't the doc start "The (upper|lower) margin ..."?

Reply:
I see a couple others as well for this tsp, but not some others.